### DISCUSSION TITLE
Suggestion for conditional display of Submissions section

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21889] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

As you know, the Submissions section is currently displayed in the WordPress dashboard > elementor. Whether we use the widget form or not. This is not really interesting.

### Describe the solution you'd like

It is better not to display the Submissions section in the wp-dashboard > elementor by default. But when we add the form widget on each of the pages and **select the Collect Submissions action**, the Submissions section is added to wp-dashboard > elementor.

I think this is a smart solution and makes the Submissions section only show if it is needed. **If you are reading this proposal, I would very much like to know what you think about it. Even if it is negative.**

thanks

**Comment by sinanisler** [comment]
"It is better not to display the Submissions section in the wp-dashboard > elementor by default. But when we add the form widget on each of the pages and select the Collect Submissions action, the Submissions section is added to wp-dashboard > elementor."

this is not clear. what do you want?
